ads/nativeroll.js

```diff
@@ -39,6 +39,14 @@ function initPlayer(data) {
     container: '#c',
     desiredOffset: 50,
     gid: data.gid,
+    onError: () => {
+      global.context.noContentAvailable();
```

Comment:
you need pass `global` object as function param

Reply:
ah, ok, did not immediately understand what you mean. Done this.